### PR TITLE
Fix BCD keys for CSS display values

### PR DESCRIPTION
**Pull request by Josh-Cena**
Fix `table_values` and `ruby_values` with the exact BCD keys

### CODE REVIEW

File: files/en-us/web/css/display-internal/index.md [hunk 1]
```diff
@@ -3,8 +3,18 @@ title: <display-internal>
...
-  - css.properties.display.table_values
-  - css.properties.display.ruby_values
+  - css.properties.display.table-row-group
+  - css.properties.display.table-header-group
+  - css.properties.display.table-footer-group
+  - css.properties.display.table-row
+  - css.properties.display.table-cell
+  - css.properties.display.table-column-group
+  - css.properties.display.table-column
+  - css.properties.display.table-caption
```
Comment (estelle) [inline files/en-us/web/css/display-internal/index.md:13]: this value is not in the https://github.com/mdn/browser-compat-data/blob/main/css/properties/display.json file. 
Should we remove it from here or add it to `display.json`?

```suggestion
```

**Review by estelle — APPROVED** [review]
You may want to remove table-caption; though we likely should have it added to bcd. Approving, with a suggestion you may want to merge or not.

File: files/en-us/web/css/display-internal/index.md (hunk 1 above)
Comment (Josh-Cena) [reply files/en-us/web/css/display-internal/index.md:13]: It's in content and is an actual value, so unless there's a clear reason I think it should be in BCD.

Comment (estelle) [reply files/en-us/web/css/display-internal/index.md:13]: i filed an issue

**Pull request merged**